BidCat/API/Auction.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ApiListener;
+using BidCat.Banks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BidCat.API
+{
+	public class Auction
+	{
+		Dictionary<string, Dictionary<int, int>> lotDict = new Dictionary<string, Dictionary<int, int>>();
+
+		AbstractBank bank;
```

Comment:
This is something we should use Dependency Injection for, rather than switch statements.